components/script/dom/webidls/ServiceWorkerContainer.webidl

```diff
@@ -6,9 +6,7 @@
 [Pref="dom.serviceworker.enabled", Exposed=(Window,Worker)]
 interface ServiceWorkerContainer : EventTarget {
   [Unforgeable] readonly attribute ServiceWorker? controller;
-  //[SameObject] readonly attribute Promise<ServiceWorkerRegistration> ready;
-
-  [NewObject, Throws] ServiceWorkerRegistration register(USVString scriptURL, optional RegistrationOptions options);
+  [SameObject] readonly attribute Promise<ServiceWorkerRegistration> ready;
```

Comment:
@jdm When running `./mach build -d`, I am seeing this error:

`WebIDL.WebIDLError: error: Unresolved type '<unresolved scope>::Promise'., /home/user/servo/components/script/dom/webidls/ServiceWorkerContainer.webidl line 9:34
  [SameObject] readonly attribute Promise<ServiceWorkerRegistration> ready;`